lib/naver/searchad/api/core/base_service.rb

```diff
@@ -1,3 +1,11 @@
+require 'addressable/uri'
+require 'addressable/template'
+require 'httpclient'
+require 'naver/searchad/api/options'
```

Comment:
I think that the general consensus for `require` vs `require_relative`, from what I understand, is:

* Use `require_relative` when it's an internal dependency

* Use `require` when it's an external dependency (because it'd be loaded in your `$:` or `$LOAD_PATH` anyway)

From my point of view, the advantage of using `require_relative` in that case is that you are 100% sure that you are loading `naver/searchad/api/options` from your own gem in that file, instead of whatever you can possibly have in `$:`

What do you think?